sources/packages/backend/libs/sims-db/src/entities/disbursement-overaward.model.ts

```diff
@@ -97,4 +100,31 @@ export class DisbursementOveraward extends RecordDataModel {
     nullable: true,
   })
   deletedAt?: Date;
+  /**
+   * Note entered during manual overaward record added.
+   */
+  @OneToOne(() => Note, { eager: false, cascade: true })
+  @JoinColumn({
+    name: "note_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  overawardNotes: Note;
```

Comment:
This should be defined as nullable since only manual records will populate it, right?